.gitignore

```diff
@@ -16,3 +16,4 @@
 coverage.txt
 codegpt
 dist
+.idea/
```

Comment:
This code patch appears to be a change to a file or project structure, rather than actual code changes. As such, there are no bugs or security vulnerabilities to review. 

However, it is important to ensure that these changes are necessary and will not cause unexpected issues or conflicts with other parts of the project. Additionally, it may be helpful to include a brief description or comment explaining the reason for the change.